src/Service.Tests/Authorization/AuthorizationResolverUnitTests.cs

```diff
@@ -1210,6 +1194,202 @@ public void ValidateClientRoleHeaderClaimIsAddedToClaimsInRequestContext()
             Assert.IsTrue(claimsInRequestContext.Count == 1);
             Assert.IsTrue(claimsInRequestContext.ContainsKey(AuthenticationOptions.ROLE_CLAIM_TYPE));
             Assert.IsTrue(TEST_ROLE.Equals(claimsInRequestContext[AuthenticationOptions.ROLE_CLAIM_TYPE].Value));
+        }*/
+
+        /// <summary>
+        /// JWT token JSON payloads may not be flat and may contain nested JSON objects or arrays.
```

Comment:
is this test method not required? why is it commented out?

Reply:
Updated to remove duplicate. but uncommented other relevant test and updated comments.